src/Illuminate/View/Factory.php

```diff
@@ -2,14 +2,15 @@
 
 namespace Illuminate\View;
 
-use Illuminate\Contracts\Container\Container;
-use Illuminate\Contracts\Events\Dispatcher;
-use Illuminate\Contracts\Support\Arrayable;
-use Illuminate\Contracts\View\Factory as FactoryContract;
 use Illuminate\Support\Arr;
+use InvalidArgumentException;
 use Illuminate\Support\Traits\Macroable;
+use Illuminate\Contracts\Events\Dispatcher;
+use Illuminate\Contracts\Support\Arrayable;
 use Illuminate\View\Engines\EngineResolver;
-use InvalidArgumentException;
+use Illuminate\Contracts\Support\Renderable;
+use Illuminate\Contracts\Container\Container;
+use Illuminate\Contracts\View\Factory as FactoryContract;
```

Comment:
the order of import is alphabetical ascending and not by length.

Reply:
Thanks crynobone. I'll adjust it when I go through the failing tests.